### PR TITLE
Add Root Me SVG

### PR DESCRIPTION
**Pull request by SiAust**
simpleicons

**Issue:** #6201
**Similarweb rank:** [113,189](https://www.similarweb.com/website/root-me.org/#overview)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I could not find a brand guide for Root Me, so chose the hex value based on the logo SVG fill colour.

[Source](https://www.root-me.org/IMG/siteon0.svg?1574327304)

### CODE REVIEW

**Comment by sachinraja** [comment]
I'm noticing a very slight difference:
image
image

You may want to try increasing the precision.

**Comment by Sanlokii** [comment]
Hi, I see the PR is still pending, how do we move this forward?
You need an SVG with a higher precision, right?

As the PR is already open, do you only need the SVG or do you have to redo the whole process following the contributing guidelines?

**Comment by adamrusted** [comment]
I've re-worked this using the source SVG again. @simple-icons/icon-review - can you take a look please? 🙏

rootme

File: icons/rootme.svg
Comment (PratyushJayachandran) [inline icons/rootme.svg:1]: image
I see this slight difference with red being the contributed SVG
rootme-compare

Comment (adamrusted) [reply icons/rootme.svg:1]: Give it another go @PratyushJayachandran

**Comment by PratyushJayachandran** [comment]
LGTM 💯 
image

**Pull request merged**